Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1

```diff
@@ -245,16 +252,29 @@
             'ApplicationPackagePathInImageStore' = $applicationPackagePathInImageStore
         }
 
+        $InstalledSdkVersion = [version](Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Service Fabric SDK" -Name FabricSDKVersion).FabricSDKVersion
+
         if ($CopyPackageTimeoutSec)
         {
-            $InstalledSdkVersion = (Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Service Fabric SDK" -Name FabricSDKVersion).FabricSDKVersion
-            if ([version]$InstalledSdkVersion -gt [version]"2.3")
+            if ($InstalledSdkVersion -gt [version]"2.3")
```

Comment:
Though the -gt works because $InstalledSdkVersion includes the build version. -ge would be easier to understand as someone can think this require atleast 2.4